crates/cubecl-runtime/build.rs

```diff
@@ -4,5 +4,6 @@ fn main() {
     // Setup cfg aliases
     cfg_aliases! {
         autotune_persistent_cache: { all(feature = "std", any(target_os = "windows", target_os = "linux", target_os = "macos")) },
+        exclusive_memory_only: { any(feature = "exclusive-memory-only", target_family = "wasm") },
```

Comment:
on wasm, exclusive memory is not a preference, but just broken if you don't use it, so better to compile it away imo.

Otherwise, maybe could also configure it with an environment variable in RuntimeOptions, like tasks_max is done, as well, but to be able to compile a cube shader you need to know whether all usages of it are spec compliant, so changing it later on doesn't really work. Kernels are re-used between clients(?) so if two are configured differently they now can't share a kernel.